aspnetcore/host-and-deploy/azure-apps/index.md

```diff
@@ -135,24 +137,48 @@ When swapping between deployment slots, any system using data protection won't b
 
 For more information, see <xref:security/data-protection/implementation/key-storage-providers>.
 <a name="deploy-aspnet-core-preview-release-to-azure-app-service"></a>
-<!-- revert this after 3.0 supported
-## Deploy ASP.NET Core preview release to Azure App Service
 
-Use one of the following approaches if the app relies on a preview release of .NET Core:
+## Deploy ASP.NET Core 3.0 to Azure App Service
 
-* [Install the preview site extension](#install-the-preview-site-extension).
+ASP.NET Core 3.0 is supported on Azure App Service. If you want to deploy preview releases of .NET Core 3.0+ in between major releases, or in the rare case during a deployment when the runtime is available but the SDK is not yet installed, use one of the techniques below to deploy an ASP.NET Core 3.0 (or later) app. 
```

Comment:
```suggestion
ASP.NET Core 3.0 is supported on Azure App Service. To deploy a preview release of a .NET Core version later than .NET Core 3.0, use one of the following techniques. These approaches are also used when the runtime is available but the SDK hasn't been installed on Azure App Service.
```

Alternatively, you could end the 2nd sentence with a colon and move the last sentence to a new paragraph (a one liner) immediately below the list as ...

> The preceding approaches are also used when the runtime is available but the SDK hasn't been installed on Azure App Service.